jitify2.hpp

```diff
@@ -1964,7 +1964,7 @@ inline bool link_programs(size_t num_programs, const std::string* programs[],
     if (linked_cubin) *linked_cubin = *programs[0];
     return true;
   }
-#if CUDA_VERSION >= 11040
+#if defined(JITIFY_ENABLE_LTO)
```

Comment:
maybe want to keep the `CUDA_VERSION >= 11040`, since it looks like a prereq for LTO from the way this is written.

Reply:
Good point. Restored the check for CUDA 11.4